deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -260,13 +267,21 @@ data:
       endpoint "#{ENV['SUMO_ENDPOINT_LOGS']}"
       verify_ssl "#{ENV['VERIFY_SSL']}"
       log_format "#{ENV['LOG_FORMAT']}"
-      flush_interval "#{ENV['FLUSH_INTERVAL']}"
-      num_threads "#{ENV['NUM_THREADS']}"
-      open_timeout 60
       add_timestamp "#{ENV['ADD_TIMESTAMP']}"
       timestamp_key "#{ENV['TIMESTAMP_KEY']}"
       proxy_uri "#{ENV['PROXY_URI']}"
+      @include buffer.output.conf
```

Comment:
This is a nice one. Can we also do some similar abstraction for other configurations? e.g.       
      data_type metrics
      metric_data_format prometheus
      disable_cookies true

Reply:
Sure I can DRY this up more.  It's too bad fluentd only supports `include` and not macro templates that we could parameterize.

Reply:
Done, can you approve again ?

Reply:
LGTM. thanks